### PR TITLE
Makes Spare more Accessible for heads (V2 because I forgot to compile the first time haha yes)

### PR DESCRIPTION
**Pull request by Tupinambis**
🆑 Tupinambis
tweak: Removes the captain's spare from the locker and places it on a table within the captain's rooms. Applies to all stations.
tweak: Unlock air alarm and door bolts for Toxins. Adds scrubber and bomb hardsuit. Just updating it to be in line with the other stations.
/🆑

[why]: As an acting captain, going through the process of grabbing an axe or ion rifle to access the captain's spare is frustrating. This change should allow heads of staff to more easily take over as acting captain, should no HoP or Captain be available.

### CODE REVIEW

**Comment by nik707** [comment]
CAN YOU NOT STEAL MY HAHA YES>?

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -82122,11 +82123,11 @@
 	},
 /area/science/explab)
 "duF" = (
-/obj/machinery/airalarm{
+/obj/effect/turf_decal/delivery,
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:82126]: ?

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -82122,11 +82123,11 @@
...
+/obj/machinery/airalarm/unlocked{
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:82127]: ??

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 3]
```diff
@@ -82122,11 +82123,11 @@
...
 	dir = 4;
-	pixel_x = -22
+	pixel_x = -23
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:82129]: ???

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 4]
```diff
@@ -89452,6 +89453,7 @@
 	dir = 4
 	},
 /obj/effect/turf_decal/bot,
+/obj/machinery/suit_storage_unit/rd,
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:89456]: ????

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 5]
```diff
@@ -99648,7 +99650,6 @@
 /area/crew_quarters/abandoned_gambling_den)
 "fpQ" = (
 /obj/machinery/door/airlock/research/glass/incinerator/toxmix_interior,
-/obj/effect/mapping_helpers/airlock/locked,
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:99651]: ?????

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 6]
```diff
@@ -100179,7 +100180,6 @@
 /area/science/mixing)
 "oIE" = (
 /obj/machinery/door/airlock/research/glass/incinerator/toxmix_exterior,
-/obj/effect/mapping_helpers/airlock/locked,
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:100182]: ??????

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 7]
```diff
@@ -100195,6 +100195,7 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 6
 	},
+/obj/machinery/portable_atmospherics/pump,
```
Comment (deathride58) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:100198]: ???????

**Review by deathride58 — APPROVED** [review]

**Pull request merged**